src/SprykerEco/Client/Unzer/UnzerClientInterface.php

```diff
@@ -38,4 +40,19 @@ public function processNotification(UnzerNotificationTransfer $unzerNotification
      * @return \Generated\Shared\Transfer\UnzerPaymentTransfer|null
      */
     public function findUpdatedUnzerPaymentForOrderAction(OrderTransfer $orderTransfer): ?UnzerPaymentTransfer;
+
+    /**
+     * Specification:
+     * - Makes Zed request.
```

Comment:
specification is not full
required properties and details are missing

Reply:
please also update specification for adjusted `performPaymentMethodsImport` facade method

Reply:
Fixed.